styles.css

```diff
@@ -151,54 +155,32 @@ body {
   align-items: flex-start;
 }
 
-.content ul {
+.langUl {
   line-height: 5px;
   list-style-type: none;
   color: #6070ff;
   margin: 24px 24px 24px 0;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
 }
 
-.content ul li {
+.langUl li {
   border-radius: 10px;
   background-color: #ebebff;
   padding: 12px;
   font-family: Poppins, sans-serif;
   font-size: 12px;
 }
 
-.content p {
-  font-family: 'Poppins', sans-serif;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  color: #344563;
-}
-
 .para {
   letter-spacing: 0.5px;
-}
-
-.content-List {
-  display: flex;
-  flex-direction: row;
   font-family: 'Poppins', sans-serif;
-  font-weight: 600;
-  font-size: 13px;
-  color: #344563;
-}
-
-.content-List p {
-  font-family: 'Poppins', sans-serif;
-  font-weight: 600;
-  font-size: 13px;
-  line-height: 54px;
-  color: #7a869a;
-  padding-right: 6px;
-}
-
-.content-List .canopy {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  padding-top: 1rem;
   color: #344563;
-  font-weight: 600;
 }
 
 .btn-1 {
```

Comment:
Great job, the buttons are working correctly 👌 But let's add some space to the bottom of the button. 
<img width="609" alt="Screenshot 2023-02-03 at 11 27 52 PM" src="https://user-images.githubusercontent.com/25887253/216704786-e4467b9e-045b-441c-ba2f-52435bdf1872.png">
<img width="609" alt="Screenshot 2023-02-03 at 11 28 12 PM" src="https://user-images.githubusercontent.com/25887253/216704791-070164c6-d6d7-4c33-88ba-99678f2d072f.png">